test/binance/ecdsa_test.go

```diff
@@ -31,18 +36,20 @@ func TestFastThresholdBinanceECDSA(t *testing.T) {
 	var signatureAlgorithms func([]*commLogger) (func(uint16) KeyGenerator, func(uint16) Signer)
 
 	verifySig = verifySignatureECDSA
-	signatureAlgorithms = ecdsaKeygenAndSign
+	signatureAlgorithms = func(loggers []*commLogger) (func(uint16) KeyGenerator, func(uint16) Signer) {
+		return ecdsaKeygenAndSign(elliptic.P256(), loggers)
```

Comment:
same thing, let's have a test case which initializes an instance with secp256k1.

Reply:
I have added a loop to iterate over the curves